lib/coins/litecoin/wallet.dart

```diff
@@ -1,7 +1,8 @@
 import 'dart:convert';
 import 'dart:typed_data';
 
-import 'package:bitcoin_base/bitcoin_base.dart';
+import 'package:bitcoin_base/bitcoin_base.dart' hide LitecoinAddress;
+import 'package:bitcoin_base/bitcoin_base.dart' as bb show LitecoinAddress;
```

Comment:
Renaming it would have been less confusing and easier for future modifications 😅 but ok

Reply:
I totally suck at naming things :p